docs/ossec_alerts.md

```diff
@@ -72,23 +71,19 @@ Next you specify the e-mail that you'll be sending alerts to, as `ossec_alert_em
 	
 Now you can move on to the SMTP and SASL settings, which are straightforward. These correspond to the outgoing e-mail address used to send the alerts instead of where you're receiving them. If that e-mail is ossec@news-org.com, the `sasl_username` would be ossec and `sasl_domain` would be news-org.com.
 
-The Postfix configuration enforces certificate verification, and requires a fingerprint to be set. You can retrieve the fingerprint of your SMTP relay by running the command below (all on one line). Please note that you will need to replace `smtp.gmail.com` and `587` with the correct domain and port for your SMTP relay.
+The Postfix configuration enforces certificate verification, and requires both a valid certificate and STARTTLS support on the SMTP relay. By default the system CAs will be used for validating the relay certificate. If you need to provide a custom CA to perform the validation, copy the cert file to `install_files/ansible-base` add a new variable to `prod-specific.yml`:
```

Comment:
The security level can be overridden during provisioning by appending `--extra-vars smtp_relay_security_level=verify` to the playbook command, or by adding that var to `prod-specific.yml`. For admins that want to adjust the security level, overriding the var will allow them to customize the environment without editing the postfix config template. Intentionally not documenting this for now, since it's quite advanced functionality, and we don't have enough data on use cases to conclude that anyone would actually take advantage of it.